infra/src/cli/render-secrets.ts

```diff
@@ -0,0 +1,80 @@
+import { generateYamlForEnv } from '../dsl/serialize-to-yaml'
+import { UberChart } from '../dsl/uber-chart'
+import { Envs } from '../environments'
+import { charts } from '../uber-charts/all-charts'
+import { SSM, ResourceGroupsTaggingAPI } from 'aws-sdk'
+import { GetResourcesOutput } from 'aws-sdk/clients/resourcegroupstaggingapi'
+
+const API_INITIALIZATION_OPTIONS = {
+  region: 'eu-west-1',
+  maxRetries: 10,
+  retryDelayOptions: { base: 200 },
+}
+
+export const renderSecrets = async (service: string) => {
+  const client = new SSM(API_INITIALIZATION_OPTIONS)
+
+  const urls: string[] = []
+  const uberChart = new UberChart(Envs.dev)
+  const services = Object.values(charts).map(
+    (chart) => generateYamlForEnv(uberChart, ...chart.dev).services,
+  )
+  const secretMap: { [name: string]: string } = {}
+  services.forEach((svc) => {
+    Object.entries(svc).forEach(([serviceName, config]) => {
+      if (serviceName == service) {
+        Object.entries(config.secrets).forEach(([envName, ssmName]) => {
+          secretMap[ssmName] = envName
+        })
+      }
+    })
+  })
+  // There seem to be only 1 parameter that's labeled as dev?
```

Comment:
@darrikonn I seem to remember you had something to do about this script at some point. Any idea why there is only 1 parameter labeled as dev now? Can we/should we simply drop that requirement?

Reply:
Yeah you're right. I wrote at least the first version of it.

There should be definitely more than 1 secret in dev labeled as dev.
Only 1 version had the label though. Might be that is what you're referring to?

Reply:
I'm mixing tags and labels :facepalm: 
Reworking this....

Reply:
Fixed now